refactor(docs): split monolithic api reference into multiple files (#722)
DisnakeDev/disnake: docs/api/webhooks.rst

.. SPDX-License-Identifier: MIT

.. currentmodule:: disnake

Webhooks

> ========
> 
> This section documents all objects related to webhooks - Discord feature which allows to send message in channels without a user\/bot account.

Reviewer: ```suggestion
========

This section documents all objects related to webhooks, a low-effort way of sending messages in channels without a user\/bot account.
```
